src/UICrafter/Program.cs

```diff
@@ -106,6 +107,20 @@
 
 builder.Services.AddHttpContextAccessor();
 
+//Snackbar settings
+builder.Services.AddMudServices(config =>
```

Comment:
This is not needed, we already do AddMudServices and I just tested it and it works without this change as it was.